packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -1,60 +1,22 @@
 import { Magento2 } from '../../../RestApi';
 import BrowserPersistence from '../../../util/simplePersistence';
 import { refresh } from '../../../util/router-helpers';
-import { getCartDetails, removeCart } from '../cart';
+import { removeCart } from '../cart';
 import { clearCheckoutDataFromStorage } from '../checkout';
 
 import actions from './actions';
 
 const { request } = Magento2;
 const storage = new BrowserPersistence();
 
-export const signIn = credentials =>
```

Comment:
Should have deleted this in the #1904 but I didn't. Doing it now since I'm trying to clean up refs to getCartDetails.

Reply:
Did you create a ticket to bring this action back? Would be nice if it referenced this comment on the off chance one of us doesn't pick it up.

You also left `SIGN_IN` in the action map, but I'm fine not touching it since it _should_ be coming back.

Reply:
I didn't -- I don't think it's necessary right now. You?

Reply:
I would probably want it moved before the next release, just so we're providing a consistent pattern for other developers to follow. Given `useSignIn` doesn't have any data fetching or effects that make our custom hooks harder to understand, doesn't have to be right now.